dds_web/commands.py

```diff
@@ -157,15 +157,15 @@ def create_new_unit(
     gc.collect()
 
 
-@click.command("update-unit")
+@click.command("update-unit-sto4")
 @click.option("--unit-id", "-u", type=str, required=True)
 @click.option("--sto4-endpoint", "-se", type=str, required=True)
 @click.option("--sto4-name", "-sn", type=str, required=True)
 @click.option("--sto4-access", "-sa", type=str, required=True)
 @click.option("--sto4-secret", "-ss", type=str, required=True)
 @flask.cli.with_appcontext
```

Comment:
again, it doesn't make sense for me to have "sto4" in all of these, because it seems the command is supposed to update all the values, which intuitively means changing the s3 provider

Reply:
Re: that all options are required -- it was because we didn't want to risk anything being left out when we were making the sto2 --> sto4 migration. 

Re: that we have `sto4` in all of these, I agree, we should change in the future but if we change it now we risk confusing ourselves in the future again. Now we have `sto2_name`, `sto4_name` etc in the database, so if we change this option in this command to `name` for example, then it will become more confusing to us later. We don't want to keep the `sto2` info in the future, so until we have removed that, we should keep as much detail as possible.